docs/faq.md

```diff
@@ -29,7 +46,7 @@ undefined.” in the logs**
 
 This usually means that EarlGrey is being linked to more than once. Ensure that only the **Test Target**
 depends on *EarlGrey.framework* and EarlGrey.framework is embedded in the app under test (i.e. *$TEST_HOST*) from the
-test target's build phase.
+test target's built products via a Copy File(s) script.
```

Comment:
> built products via a Copy File(s) script.

is not correct. Xcode refers to `Build Phases`

<img width="465" alt="build_phases_fail" src="https://cloud.githubusercontent.com/assets/1173057/24107653/b3275a82-0d61-11e7-9358-b3b7e3799fca.png">